### PR TITLE
ci: add release 3.5 image details to the mirror image list

### PR DESCRIPTION
**Pull request by humblec**
The release 3.5 image details were missing and this commit add the
same.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@nixpanic not sure how ci was passing without the 3.5 entries in the release 3.5 branch. 

Also one thing noted here is, this image references images from docker hub instead of quay.io . If that need to be changed as well, please mention, I can add the same to this PR.

**Comment by humblec** [comment]
> @nixpanic not sure how ci was passing without the 3.5 entries in the release 3.5 branch.
> 
> Also one thing noted here is, this image references images from docker hub instead of quay.io . If that need to be changed as well, please mention, I can add the same to this PR.

I have added that too as a second commit..

**Comment by humblec** [comment]
> > @nixpanic not sure how ci was passing without the 3.5 entries in the release 3.5 branch.
> > Also one thing noted here is, this image references images from docker hub instead of quay.io . If that need to be changed as well, please mention, I can add the same to this PR.
> 
> I have added that too as a second commit..

In the scripts of CI its referring completely to `docker.io` , may be its kept that way to be in parity with `rook` images?

File: mirror/images.txt [hunk 1]
```diff
@@ -9,12 +9,18 @@
 # ceph-csi devel
 docker.io/rook/ceph:v1.8.2	rook/ceph:v1.8.2
 
+
```
Comment (nixpanic) [inline mirror/images.txt]: please remove the additional empty lines. Just a single empty line between section suffices.

File: mirror/images.txt [hunk 2]
```diff
@@ -9,12 +9,18 @@
...
+# ceph-csi v3.5
+quay.io/cephcsi/ceph:v16	ceph/ceph:v16
```
Comment (nixpanic) [inline mirror/images.txt]: this looks incorrect, we do not use `quay.io/cephcsi/`, the images are located under `quay.io/ceph/`?

File: mirror/images.txt [hunk 3]
```diff
@@ -9,12 +9,18 @@
...
+docker.io/rook/ceph:v1.6.2	rook/ceph:v1.6.2
+
+
 # ceph-csi v3.4
-docker.io/ceph/ceph:v16		ceph/ceph:v16
+quay.io/cephcsi/ceph:v16	ceph/ceph:v16
```
Comment (nixpanic) [inline mirror/images.txt]: there is no need to duplicate things, that will probably cause images to pushed to the CI mirror multiple times (and overwrites), so it just causes extra network traffic.

**Review by nixpanic — CHANGES_REQUESTED** [review]
> In the scripts of CI its referring completely to `docker.io` , may be its kept that way to be in parity with `rook` images?

Ceph did not push to Quay.io when we started to use the mirroring. Docker Hub has the pull restrictions, Quay.io does not.

File: mirror/images.txt (hunk 3 above)
Comment (humblec) [reply mirror/images.txt]: sure, removed duplicated instances too @nixpanic

File: mirror/images.txt (hunk 1 above)
Comment (nixpanic) [reply mirror/images.txt]: you marked this as resolved, but I still see the additional empty lines in this commit?

File: mirror/images.txt [hunk 4]
```diff
@@ -9,18 +9,12 @@
...
 
-
 # ceph-csi v3.5
-docker.io/ceph/ceph:v16		ceph/ceph:v16
-docker.io/rook/ceph:v1.6.2	rook/ceph:v1.6.2
-
-
-# ceph-csi v3.4
-docker.io/ceph/ceph:v16		ceph/ceph:v16
+quay.io/ceph/ceph:v16		ceph/ceph:v16
```
Comment (nixpanic) [inline mirror/images.txt:14]: please keep the values aligned (with tabs)

File: mirror/images.txt [hunk 5]
```diff
@@ -8,6 +8,8 @@
 
 # ceph-csi devel
 docker.io/rook/ceph:v1.8.2	rook/ceph:v1.8.2
+quay.io/ceph/ceph:v17		ceph/ceph:v17
+
```
Comment (nixpanic) [inline mirror/images.txt:12]: again an unneeded empty line, please remove those.

**Review by nixpanic — CHANGES_REQUESTED** [review]

File: mirror/images.txt (hunk 4 above)
Comment (humblec) [reply mirror/images.txt:14]: not sure there is an inconsistency with tabs , however it looks to be rendering correctly to me https://github.com/ceph/ceph-csi/blob/511ae20d106ff5e3fd08ec2ed533e9b6e357c555/mirror/images.txt

Comment (nixpanic) [reply mirror/images.txt:14]: thanks, seems the GitHub webui/PR does not show it correctly for some reason.

**Review by nixpanic — APPROVED** [review]
The commits still add the extra empty lines, and a next one removes them. Not very clean, but the result looks good now.

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**